src/WebJobs.Script/Binding/ExtensionLoader.cs

```diff
@@ -1,228 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.Azure.WebJobs.Host.Config;
-using Microsoft.Azure.WebJobs.Script.Config;
-using Microsoft.Azure.WebJobs.Script.Description;
-using Microsoft.Extensions.Logging;
-
-namespace Microsoft.Azure.WebJobs.Script.Binding
-{
-    internal class ExtensionLoader
```

Comment:
It looks like in this branch, you reverted/missed a bunch of refactorings I did in this commit: https://github.com/Azure/azure-webjobs-sdk-script/commit/0bf733e1ab3e1c3137c917c2dc16964a0d7fbc4e#diff-b3a5125652975b25753e4ead9265dbf9. Many of the changes I made in that commit look like they weren't merged.

Reply:
Yes, there were A LOT of conflicts with that refactoring. I'll need to pull that in but needed to keep this as it had change pretty significantly from the version you modified and with our preview/target dates, we couldn't afford to regress the new logic.